src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java

```diff
@@ -202,9 +202,10 @@ private static boolean containerVersion(String version){
      * @throws LibertyException
      */
     public static String getMavenSettingsCmd(Project project, VirtualFile buildFile) throws LibertyException {
+        String wrappedMaven = "Use Maven wrapper";
```

Comment:
Was `MavenServerManager.WRAPPED_MAVEN` removed? Does this have the same value? Perhaps make this a static final constant.

Reply:
Made static final.